samples/apps/smallbank/smallbank.cmake

```diff
@@ -14,17 +14,10 @@ sign_app_library(
   ${CCF_DIR}/src/apps/sample_key.pem
 )
 
-if(${SERVICE_IDENTITY_CURVE_CHOICE} STREQUAL "secp256k1_bitcoin")
-  set(SMALL_BANK_SIGNED_VERIFICATION_FILE
-      ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank_50k.json
-  )
-  set(SMALL_BANK_SIGNED_ITERATIONS 50000)
-else()
-  set(SMALL_BANK_SIGNED_VERIFICATION_FILE
-      ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank_2k.json
-  )
-  set(SMALL_BANK_SIGNED_ITERATIONS 2000)
-endif()
+set(SMALL_BANK_SIGNED_VERIFICATION_FILE
```

Comment:
We now use the fast curve for all users' identities in the client signature perf tests. Should we default to 50,000 rather than 2,000?

Reply:
Maybe - not sure how reasonable this is for PBFT. I've made the change, will see what the CI looks like on this PR.

In a related change I've added a function to construct the `_VERIFICATION_FILE` from the `ITERATIONS`, so we don't have to repeat ourselves so often.

Reply:
I'd think longer is better/more accurate, as long as it doesn't take more than 1 minute to run the test in the CI?

Reply:
`50003 transactions took 15747ms`

15s sounds reasonable, so I think 50k for all the signed tests is good.